tests/test_object_detection.py

```diff
@@ -388,6 +387,15 @@ def test_badloc_score_shifts_in_correct_direction():
     assert scores[1] > scores[2]
 
 
+def test_badloc_scores_indexed_correctly():
+    # test badloc scores indexed correctly when len(idx_at_least_low_probability_threshold) < len(idx_at_least_intersection_threshold)
+    low_prob = 0.2
+    prediction = copy.deepcopy(predictions[0])
+    prediction[3][1][-1] = low_prob  # artificially set low probability for box in class. 1 < 2
+    label = copy.deepcopy(labels[0])
+    _ = compute_badloc_box_scores(labels=[label], predictions=[prediction])
+
```

Comment:
there's nothing to be asserted in this test?

Reply:
are you sure there doesn't need to be analogous unit tests for the swap / overlook issue subtypes?